cluster/gce/windows/smoke-test.sh

```diff
@@ -588,21 +573,25 @@ function test_windows_pod_to_k8s_service {
   local service_address="$service_ip:$service_port"
 
   echo "curl-ing $service address from Windows pod: $service_address"
-  # Performing a web request to the heapster service results in an expected 404
-  # response; this code snippet filters out the expected 404 from other status
-  # codes that indicate failure.
+  # curl-ing the metrics-server service results in a ServerProtocolViolation
```

Comment:
what about curling default-http-backend?

Reply:
In `kubectl get services --all-namespaces`, default-http-backend has type `NodePort` while metrics-server has type `ClusterIP`, so I chose metrics-server because I think heapster also had type `ClusterIP`. Do you know if it matters one way or the other for the service connectivity testing we're trying to do here?

Reply:
NodePort has everything 'ClusterIP' has, and some extra. For testing connectivity with cluster ip, using NortPort is the same as using 'ClusterIP' (https://i.stack.imgur.com/1owA5.jpg).

Reply:
curl-ing the default-http-backend from a Linux pod in my cluster always seems to timeout. Any idea why?

```
$ kubectl exec linux-ubuntu-79d5cd56f5-sbgcm -- curl -m 20 "http://10.0.66.218:80"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:20 --:--:--     0
curl: (28) Connection timed out after 20001 milliseconds
command terminated with exit code 28
```

I get the same result when I just plug in a random IP address. Is this a bug in our cluster setup or is there a reason default-http-backend doesn't work?

Reply:
I suspect I know why my curl command was timing out in my previous comment. In a new cluster I'm now getting 404s from default-http-backend, which I think is expected.

Anyway, I'd like to just leave this as-is (curl-ing from metrics-server rather than default-http-backend); neither works like a typical web server, and the code in this PR is working well enough for our smoke test.